concordium-node/src/bin/collector.rs

```diff
@@ -188,18 +169,12 @@ async fn main() {
                     }
                 }
                 Err(e) => {
-                    let _ = atomic_counter.fetch_add(1, AtomicOrdering::SeqCst);
                     error!(
                         "gRPC failed with \"{}\" for {}, sleeping for {} ms",
                         e, &grpc_host, conf.collector_interval
                     );
                 }
             }
-
-            if grpc_failure_count + 1 >= conf.max_grpc_failures_allowed as usize {
-                error!("Too many gRPC failures, exiting!");
-                exit(1);
-            }
         }
         trace!("Sleeping for {} ms", conf.collector_interval);
```

Comment:
I know this is from before already, but I think because we use `tokio`, it would be better to use the [Interval](https://docs.rs/tokio/latest/tokio/time/struct.Interval.html) instead of `std::thread::sleep`.